nsv13/game/custom_turfs.dm

```diff
@@ -36,7 +60,31 @@
 			transfer_fingerprints_to(T)
 			qdel(src)
 			return
-	. = ..()
+
+	else //Build false wall
+		if(S.get_amount() < 2)
+			to_chat(user, "<span class='warning'>You need two sheets of [S] to create a false wall!</span>")
+			return
+
+		to_chat(user, "<span class='notice'>You start building a false wall with [S]...</span>")
+		if (do_after(user, 40, target = src))
+			if(S.get_amount() < 2)
+				return
```

Comment:
Add in a warning here that you need the two sheets otherwise the person might not know if they are making more than one false wall at a time.